extras/onewire/onewire.h

```diff
@@ -0,0 +1,163 @@
+#ifndef __ONEWIRE_H__
+#define __ONEWIRE_H__
+
+#include <espressif/esp_misc.h> // sdk_os_delay_us
+#include "FreeRTOS.h"
+
+// 1 for keeping the parasitic power on H
+#define owDefaultPower 1
+
+#define NUM_OW 20
+#define FALSE 0
+#define TRUE 1
+
+// You can exclude certain features from OneWire.  In theory, this
```

Comment:
We have a lot more resources than an 8-bit micro, so can we remove these compile-time switches and just leave this features enabled by default?